packages/select/src/select.vue

```diff
@@ -37,7 +37,7 @@
                 disable-transitions
                 @close="deleteTag($event, selected[0])"
               >
-                <span class="el-select__tags-text">{{ selected[0].currentLabel }}</span>
+                <span class="el-select__tags-text" :style="{ 'text-overflow':'ellipsis',verticalAlign: 'bottom','max-width': inputWidth - 120 + 'px', 'display':'inline-block','overflowX':'hidden' }">{{ selected[0].currentLabel }}</span>
```

Comment:
Writing this into a variable and bind it, would've been more clean. Also, add tests against it.